sdk/documentintelligence/test-resources.json

```diff
@@ -89,216 +72,80 @@
                 "signedResource": "c"
             }
         },
-        "multiPageTestingDataContainer": {
```

Comment:
are these not necessary anymore?

Reply:
I don't see it's being used anywhere for now. We can add it whenever we want.